Tasks/InstallSSHKeyV0/task.json

```diff
@@ -29,7 +29,7 @@
             "aliases": [
                 "knownHostsEntry"
             ],
-            "type": "string",
+            "type": "multiLine",
```

Comment:
Could you please update related description strings? It doesn't look precise now since doesn't describe ability to define several keys

Reply:
Added extended description